lib/publisher.rb

```diff
@@ -0,0 +1,44 @@
+class Publisher
+  def self.publish(presenter)
+    new(presenter).publish
+  end
+
+  attr_reader :presenter
+
+  def initialize(presenter)
+    @presenter = presenter
+  end
+
+  def publish
+    response = send_item_to_draft_stack
+    response = publish_links if presenter.links && response_successful?(response)
+    response = publish_item if response_successful?(response)
+    response
```

Comment:
It feels like the caller of this shouldn't need to know about the response, and understand that a 200 is good and other things are bad. Looking through the rest of the PR, this return value isn't used for anything, what error handling needs to happen here?